lms/djangoapps/class_dashboard/views.py

```diff
@@ -113,11 +115,12 @@ def section_problem_grade_distrib(request, course_id, section, enrollment):
     course_key = SlashSeparatedCourseKey.from_deprecated_string(course_id)
     if has_instructor_access_for_class(request.user, course_key):
         try:
-<<<<<<< HEAD
-            json = dashboard_data.get_d3_section_grade_distrib(course_key, section, int(enrollment))
-=======
-            data = dashboard_data.get_d3_section_grade_distrib(course_key, section)
->>>>>>> 00b75f0119b981641833240be214ef2076329747
+# TODO:FUNK <<<<<<< HEAD
+#             json = dashboard_data.get_d3_section_grade_distrib(course_key, section, int(enrollment))
+            data = dashboard_data.get_d3_section_grade_distrib(course_key, section, int(enrollment))
+# TODO:FUNK =======
+#             data = dashboard_data.get_d3_section_grade_distrib(course_key, section)
+# TODO:FUNK >>>>>>> 00b75f0119b981641833240be214ef2076329747
```

Comment:
@dylanrhodes I think you had originally written these blocks; does this conflict resolution look right?